manifest.konnector

```diff
@@ -22,11 +22,18 @@
       "openid+toutatice+offline_access"
     ]
   },
+  "folders": [{"defaultDir": "$Settings/$Home"}],
   "data_types": [
     "contacts"
   ],
   "screenshots": [],
   "permissions": {
+    "files": {
+      "type": "io.cozy.files",
+      "verbs": [
+        "ALL"
+      ]
+    },
```

Comment:
You should not give to the konnector ALL the right to the io.cozy.files doctype. 

You should only give : 
- POST to create
- PUT to edit 

That's all.

Hmm, it seems that saveFiles can destroy a file https://github.com/konnectors/libs/blob/07d63ee54a15717b5f918dbfd47133e789a9140e/packages/cozy-konnector-libs/src/libs/saveFiles.js#L627 so you should add DELETE too.

Also, since the konnector is scoped to `/Settings/Home` you should only give the permission to this path.See https://docs.cozy.io/en/cozy-stack/permissions/#selector and https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.files.md#folder to know the field to use ;)

Reply:
Changes incoming !